### PR TITLE
Features/hide api key

### LINKED ISSUE #20: Repository contains API key
You should not store API keys in your repository. See for example: https://www.freecodecamp.org/news/how-to-securely-store-api-keys-4ff3ea19ebda/

### PR DESCRIPTION
**Pull request by laytonmaes**
<h1>PULL REQUEST</h1>

<ul>
  <li>What does it change?</li>
    makes api key reliant on local untracked js files for temporary security
  <li>What does it fix?</li>
   makes api key anonymous instead of public.
</ul>

<h2>Type of Change</h2>

_Delete options that are not relevant_

- [x] **Bug Fix**

<h2>How has this been tested?</h2>

- [x] **JS**
- [x] **Dev Tools**
- [x] **Website**

<h2>Checklist</h2>

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new bugs
- [x] I have tagged project partners for review & it has been reviewed 

<h2>Does this PR close any issues?</h2>
close #20

### CODE REVIEW

File: .API_KEY_JS_sample [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+// const API_KEY = "Your_API_"
```
Comment (laytonmaes) [inline .API_KEY_JS_sample:2]: should be fixed to reflect actual ignored file

**Review by Neil-B-Linton — APPROVED** [review]
This looks amazing, you did an awesome job working with hiding this environment to keep our top secrets-secret. Good job with the self-catch!

File: Views/Dashboard.js [hunk 2]
```diff
@@ -27,10 +29,11 @@ export default function Dashboard() {
     getView(uri)
   }
 
+  console.log("env key", API_KEY)
 
   const handleSubmit = (uri) => {
     const data = {
-        api_key: 'NklRmoWXEJSh0KJwF8SHOJYEwpX5FnSZyr7n5bOJ5nrwkpEHKz',
+        api_key: API_KEY,
         images: [uri[0]],
```
Comment (BlueNealis) [inline Views/Dashboard.js:37]: Secret KEY secret Key

File: package.json [hunk 3]
```diff
@@ -18,6 +19,8 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-native": "0.68.2",
+    "react-native-dotenv": "^3.3.1",
+    "react-native-env": "^0.0.6",
     "react-native-paper": "^4.12.3",
```
Comment (BlueNealis) [inline package.json:24]: Oh new packages? I feel like im sitting by the mailbox waiting for them

File: Views/Dashboard.js [hunk 4]
```diff
@@ -27,10 +29,11 @@ export default function Dashboard() {
     getView(uri)
   }
 
+  console.log("env key", API_KEY)
```
Comment (BlueNealis) [inline Views/Dashboard.js:33]: We'll have to remember to remove this lil baby console log

**Review by BlueNealis — APPROVED** [review]
Yay thanks for implementing this

**Pull request merged**